packages/base-driver/test/basedriver/driver-tests.js

```diff
@@ -55,6 +55,12 @@ function baseDriverUnitTests (DriverClass, defaultCaps = {}) {
       sessId.length.should.be.above(5);
     });
 
+    it('should setup log prefix after createSession', async function () {
+      const previousPrefix = d.log.prefix;
+      await d.createSession(null, null, w3cCaps);
+      previousPrefix.should.not.eql(d.log.prefix);
+    });
```

Comment:
Don't we need to reset the prefix after `deleteSession` is called?  Are we doing that?  If not, we need to do that, I think. And add a test.

It'd be more valuable to make an assertion about the expected value (since there's logic involved in determining the prefix) instead of just asserting it has changed (in the above test case and also one for `deleteSession`)

Reply:
updated the test